src/ankh/cmd/ankh/main.go

```diff
@@ -156,6 +157,11 @@ func main() {
 			Desc:   "The data directory for ankh template history",
 			EnvVar: "ANKHDATADIR",
 		})
+		helmSet = app.Strings(cli.StringsOpt{
+			Name:  "set",
+			Desc:  "Global variables passed to helm with helm --set, will override variables set in ankhconfig global",
```

Comment:
s/helm/ankh?

Reply:
we could probably clarify by saying `directly to Helm via --set` since indeed these are values passed to Helm

Reply:
er yeah the wording isn't quite as helpful as it should be here, probably should add format as well